### ISSUE TITLE
Evaluation PPL, starts going up after 10 epochs

### ISSUE THREAD

**Issue by GlazeDonuts** [bentrevett/pytorch-seq2seq#102]
I implemented your notebooks (upto the 4th one) with a few changes but the validation loss keeps going up after around 10 epochs. Also, just to be sure, I simply ran your notebook on colab, and it is the same thing, any idea as to why is this happening? I cannot stop training at just 10 epochs because the losses are too high (around 1.5+) and if I keep going, it is clearly overfitting on the training data. So, what shall I do?
I have attached a screenshot showing how the val. PPL rises after a few epochs
![Screenshot from 2020-04-23 07-42-35](https://user-images.githubusercontent.com/27892015/80052248-dfa43e80-8537-11ea-8778-4dbcf0030c69.png)

**Comment by bentrevett** [comment]
Seems like your model is overfitting. One thing to try is regularization, e.g. increasing dropout or using `weight_decay` on the optimizer. 

Another thing to look into that isn't mentioned in the tutorials is increasing the default `teacher_forcing_ratio` - try setting it to 0.75 and if that improves performance then try increasing it to 1. This will actually cause your validation losses to be higher, but your BLEU scores will end up being better.
